assets/css/whyChooseUs.css

```diff
@@ -0,0 +1,59 @@
+
+.whychooseus-card-container {
+    display: flex;
+    gap: 55px;
+    align-items: center;
+    justify-content: center;
+    margin-left: 20%;
+    margin-right: 20%;
+    flex-wrap: wrap;
+}
+
+.whychooseus-page-heading {
+    font-size: 25px;
+    font-weight: bolder;
+    text-align: center;
+    justify-content: center;
+    font-family: system-ui;
+}
+
+.whychooseus-card{
+    /*display: flex;*/
```

Comment:
If display flex is not added to the class align-items, justify-content, and other things related to flex will not work, if not needed remove them @kajalwakchaure18